config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -187,3 +187,51 @@ periodics:
           limits:
             cpu: "2"
             memory: "6Gi"
+
+  # This jobs runs e2e.test with a focus on long running tests for all alpha/beta storage features on a kind cluster
+  - name: pull-kubernetes-e2e-storage-kind-alpha-beta-features-slow
+    cluster: eks-prow-build-cluster
+    optional: true
+    always_run: false # manual for testing new configurations
+    decorate: true
+    interval: 12h
+    skip_branches:
+    - release-\d+\.\d+
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    path_alias: k8s.io/kubernetes
+    annotations:
+      testgrid-dashboards: presubmits-kubernetes-nonblocking
+      testgrid-tab-name: pull-kubernetes-e2e-storage-kind-alpha-beta-features-slow
```

Comment:
the testgrid tab name should also be updated (or it can be automatically determined from the job name IIRC)